packages/muelu/research/regionMG/test/structured/Driver_Structured_Regions.cpp

```diff
@@ -539,21 +539,20 @@ int main_(Teuchos::CommandLineProcessor &clp, Xpetra::UnderlyingLib& lib, int ar
 
   RCP<TimeMonitor> tmLocal = rcp(new TimeMonitor(*TimeMonitor::getNewTimer("Driver: 3.1 - Build Region Maps")));
 
-  const int maxRegPerProc = 1;
-  std::vector<Teuchos::RCP<Xpetra::Map<LO,GO,NO> > > rowMapPerGrp(maxRegPerProc), colMapPerGrp(maxRegPerProc);
-  std::vector<Teuchos::RCP<Xpetra::Map<LO,GO,NO> > > revisedRowMapPerGrp(maxRegPerProc), revisedColMapPerGrp(maxRegPerProc);
-  rowMapPerGrp[0] = Xpetra::MapFactory<LO,GO,Node>::Build(dofMap->lib(),
+  Teuchos::RCP<Xpetra::Map<LO,GO,NO> > rowMapPerGrp, colMapPerGrp;
+  Teuchos::RCP<Xpetra::Map<LO,GO,NO> > revisedRowMapPerGrp, revisedColMapPerGrp;
+  rowMapPerGrp = Xpetra::MapFactory<LO,GO,Node>::Build(dofMap->lib(),
```

Comment:
The removed `std::vector` was related to the concept of "groups". Hence, this could be renamed to 
```suggestion
  Teuchos::RCP<Xpetra::Map<LO,GO,NO> > rowMap, colMap;
  Teuchos::RCP<Xpetra::Map<LO,GO,NO> > revisedRowMap, revisedColMap;
  rowMapPerGrp = Xpetra::MapFactory<LO,GO,Node>::Build(dofMap->lib(),
```